### ISSUE TITLE
Run GSLAM on webcam or RGB-D camera?

### ISSUE THREAD

**Issue by linhtn** [zdzhaoyong/GSLAM#43]
Hello @zdzhaoyong,
Thank for your great contribution. I want to ask you a question.
Can GSLAM run directly on webcam or RGB-D camera?

**Comment by zdzhaoyong** [comment]
Yes, of course. GSLAM already implemented dataset plugin 'cvmono' to support webcam, for RGBD camera, please implement you own dataset plugin. See: https://zdzhaoyong.github.io/GSLAM/dataset.html
